frontend/src/features/projects/assess/steps/ReviewApproveStep.tsx

```diff
@@ -98,6 +98,7 @@ const ReviewApproveStep = ({ formikRef }: IStepProps) => {
         innerRef={formikRef}
         onSubmit={(values: IProject) => {
           const workflowCode = getNextWorkflowCode(submitStatusCode, values);
+          if (values.rowVersion !== project.rowVersion) values.rowVersion = project.rowVersion;
```

Comment:
Should be able to fix this by setting enablereinitialize to true on the <Formik> component within the ReviewApproveStep.tsx

Reply:
Ooh, thank you - works!